src/index.js

```diff
@@ -0,0 +1,4 @@
+import './style.css';
+import loadEvents from './modules/loadevents.js';
```

Comment:
- [ ] Nice job for importing a new JavaScript file as a module. For us to fulfill the requirements of this project, the newly imported file should contain methods related to the status updates (`completed` : `true` / `false`). However, this file does not contain methods related to the status update.  This file is basically representing nothing for now. Please make sure to fix this issue and move a step forward towards approval.